packages/DropdownMenu/stories/examples/DropdownMenuExample.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { select } from "@storybook/addon-knobs";
+import L10n from "@paprika/l10n";
+import DropdownMenu from "../../src";
+
+const DropdownMenuExample = () => {
+  const handleConfirm = handleCloseMenu => {
+    handleCloseMenu();
+  };
+
+  const handleCancel = handleCloseMenu => {
+    handleCloseMenu();
+  };
+
+  return (
+    <L10n locale={select("locale", ["en", "de", "es", "fr", "ja", "pt", "zh"], "en")}>
+      <DropdownMenu
+        align="bottom"
+        renderTrigger={({ isOpen, handleOpenMenu }) => (
+          <DropdownMenu.Trigger isOpen={isOpen} handleOpenMenu={handleOpenMenu}>
+            Trigger
+          </DropdownMenu.Trigger>
+        )}
+      >
+        <DropdownMenu.Item onClick={() => {}}>Edit</DropdownMenu.Item>
+        <DropdownMenu.Item onClick={() => {}}>Duplicate</DropdownMenu.Item>
+        <DropdownMenu.Item isDestructive isDisabled onClick={() => {}}>
+          Google
+        </DropdownMenu.Item>
+        <DropdownMenu.Item isLink onClick={() => {}}>
+          Is Link Item
+        </DropdownMenu.Item>
+        <DropdownMenu.Item isDisabled onClick={() => {}}>
+          Galvanize
+        </DropdownMenu.Item>
+        <DropdownMenu.Divider />
+        <DropdownMenu.Item
+          isDestructive
+          renderConfirmation={handleCloseMenu => {
```

Comment:
Not a big deal, especially for story code, but I'd prefer `onCloseMenu` (or even `onClose`) for this prop.  It's not defined here, so I wouldn't use a `handle` prefix.  I _would_ use it for `handleConfirm` and `handleCancel`, like you are doing... _except_ I think you can use use this `onClose` prop directly.